src/state-mgmt/reducers.ts

```diff
@@ -64,10 +66,31 @@ const user = (state = { ...initialState.user }, action) => {
 const env = () => {
   return environment;
 };
+
+/*
```

Comment:
This shall not be there

Reply:
Hi @saurshaz 

Should I remove the env part ?

Reply:
you can , if it is not used